lldb/test/API/lang/swift/private_member/TestSwiftPrivateMember.py

```diff
@@ -0,0 +1,18 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+
+class TestSwiftPrivateMember(TestBase):
+
+    @skipUnlessDarwin
+    @swiftTest
+    def test(self):
+        self.build()
+
+        target,  process, thread, _ = lldbutil.run_to_source_breakpoint(
+            self, "break here", lldb.SBFileSpec('main.swift'))
+        frame = thread.frames[0]
+        x = frame.FindVariable("x")
+        member = x.GetChildAtIndex(0)
```

Comment:
is there a reason for using child index 0 vs child member name`"member"`? or is it all the same either way?

Reply:
because we want to trigger GetChildCompilerTypeAtIndex()